src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -379,7 +379,7 @@ public SectionDetailsBundle getSectionForCourse(String section, String courseId)
         for (int i = 0; i < students.size(); i++) {
             StudentAttributes s = students.get(i);
             
-            if (section == null) {   // First student of first section
+            if (section == null) { // First student of first section
```

Comment:
Might be better to move the comments to after the `if`, but I'm not exactly sure on this.

On the one hand this comment describes what the condition checks for instead of what goes on if the condition is true, but putting it on the line after the condition may be the only maintainable way to do this since all other locations would be out of place.

Eg. "out of place" areas:
Before conditional:
```
// this checks for whether the user is logged in
} else if (user.isLoggedIn()) {
```
kind of breaks the position of the code.

On same line as conditional
``` } else if (user.isLoggedIn()) { // this checks for whether the user is logged in```
may not work if the comment needs more than a single line

Perhaps this might require some discussion, but I'm okay if you leave these as is.